help_to_heat/frontdoor/views.py

```diff
@@ -301,18 +301,19 @@ def get_context(self, request, session_id, *args, **kwargs):
         return {"council_tax_band_options": council_tax_bands}
 
     def handle_post(self, request, session_id, page_name, data, is_change_page):
-        return super().handle_post(request, session_id, page_name, data, is_change_page)
+        session_data = interface.api.session.get_session(session_id)
```

Comment:
So this logic is now happening in the handle_post for the `council tax` page. I'm admittedly not a fan of it this approach, but it means the EPC is being calculated before the council tax page redirects anywhere, so it'll redirect to the right page.

Reply:
(I am very open to a better place to put this logic. I couldn't think of anywhere but hopefully someone else can? :D)

Reply:
I think the ideal version of this is for the EPC page to worry about this, and redirect if required, but as discussed that requires some additional routing/handler infrastructure. This is fine for now.